CONTRIBUTING.md

```diff
@@ -96,6 +96,12 @@ Here are the instructions for running Firefox from the command line (MacOS + Fir
 $ /Applications/Firefox.app/Contents/MacOS/firefox-bin --start-debugger-server 6080 -P development
 ```
 
+**note** - In windows you should replace --start-debugger-server with -start-debugger-server:
```

Comment:
Can you put a header up for each OS instead of this line?

Something like this:

**MacOS**
```
/App...
```

**Windows**
```
C:\Program Files...
```